dataactvalidator/scripts/loadFile.py

```diff
@@ -144,6 +144,15 @@ def loadSF133(filename, fiscal_year, fiscal_period, force_load=False):
              "agency_identifier": {"pad_to_length": 3},
              "main_account_code": {"pad_to_length": 4},
              "sub_account_code": {"pad_to_length": 3},
+             # next 3 lines handle the TAS fields that shouldn't
+             # be padded but should still be empty spaces,
+             # at least temporarily, so that downstream formatting
+             # and cleaning functions work correctly. this works
+             # around the fact that sometimes the incoming data
+             # is a single space and sometimes it's blank.
+             "beginning_period_of_availa": {"pad_to_length": 0},
+             "ending_period_of_availabil": {"pad_to_length": 0},
+             "availability_type_code": {"pad_to_length": 0},
```

Comment:
Should these explicitly specify the 'keep_null' option, since that seems to be the handling we're trying to get here?  More generally, since stripping extra whitespace and converting blanks to nulls is something we wanted to always do independently of padding, should this functionality happen in LoaderUtils.cleanData before the options are applied?  FieldCleaner.cleanRow does this for the values in the files provided by users, so we may want to apply the same handling to domain value loading to ensure they match.